actor/builtin/miner/miner.go

```diff
@@ -516,6 +519,25 @@ func (ma *Actor) GetPoStState(ctx exec.VMContext) (*big.Int, uint8, error) {
 	return big.NewInt(result), 0, nil
 }
 
+// GetSlashable returns whether this miner is slashable. At this time this means only that
```

Comment:
This name is a bit misleading.  The comment does a good job of explaining it but if I only saw the name I would for sure think this returns true when the miner actor can actually be slashed.  Because this returns true when there are sectors in the miner's set what about `GetProving`, `Proving`, `ActivelyProving` etc, something along those lines?